src/libostree/ostree-sign-x509.h

```diff
@@ -0,0 +1,56 @@
+/* vim:set et sw=2 cin cino=t0,f0,(0,{s,>2s,n-s,^-s,e2s: */
+
+/*
+ * Copyright © 2019 Collabora Ltd.
```

Comment:
I think you can add a different copyright here

Reply:
Fixed.